inference-engine/thirdparty/clDNN/tests/test_cases/batch_to_space_gpu_test.cpp

```diff
@@ -54,7 +54,7 @@ TEST(batch_to_space_fp16_gpu, i8111_bs1222_cb0000_ce0000) {
     ASSERT_EQ(output_ptr.size(), expected_results.size());
 
     for (size_t i = 0; i < expected_results.size(); ++i) {
-        EXPECT_EQ(expected_results[i], float16_to_float32(output_ptr[i]));
```

Comment:
Why? This seems to be illegal change as expected_results has `float` type, while `output_ptr` is uint16_t (which represents half)

Reply:
reverted. I thought that `output_ptr` of internal type `FLOAT16` which can be automatically converted to `float`